### ISSUE TITLE
[Dropdown] Multiple level dropdown should remain closed on selecting submenu item on IE11

### ISSUE THREAD

**Issue by SandeepThomas** [Semantic-Org/Semantic-UI#6565]
Issue on IE11.

### Steps
Select an item from submenu. 

### Expected Result
Dropdown should select the value, close and remain closed.

### Actual Result
Dropdown opens again if submenu item is selected.

### Version
2.3.3

### Testcase
https://jsbin.com/yofawovahi/edit?html,js,output

**Comment by KADlancer** [comment]
Yeah, I found the same problem both in Browserstack and on a real PC/ie11.

**Comment by jlukic** [comment]
I can reproduce. IE11 is refocusing the dropdown for some reason and it looks identical to a normal click.

I couldn't find any quick fix, but there's probably something i can do.

**Comment by lubber-de** [comment]
Fixed since 2.6.4
https://github.com/fomantic/Fomantic-UI/pull/247
